### ISSUE TITLE
mw:73297

### ISSUE THREAD

**Issue by drdhaval2785** [sanskrit-lexicon/csl-orig#50]
date:	09/27/2019 14:58:52
dict:	mw
Lnum:	73297
hw:	ci
old:	Desid. Caus.
new:	has to be in bold characters; Pass. also is not and should be
comm:	
email:	Caujolle

**Comment by funderburkjim** [comment]
The markup, in MW, of
`<div n="vp"/><ab>Caus.</ab>`  has the effect in displays of making bold **Caus.**.
This is due to basicadjust.php

The line relative to the correction submission for 'ci' is, before and after change:
```
old:
<div n="to"/><ab>Desid.</ab> <ab>Caus.</ab> (<ab>p.</ab> <s>cicIzayat</s>) to cause any one to wish to arrange in order, <ls>Bhaṭṭ. iii, 33</ls> :
new:
<div n="vp"/><ab>Desid. Caus.</ab> (<ab>p.</ab> <s>cicIzayat</s>) to cause any one to wish to arrange in order, <ls>Bhaṭṭ. iii, 33</ls> :
```

There are two things keeping 'Desid. Caus.' from displaying in Bold:
* the div type is 'to', rather than 'vp'.  
   * Note 1: 'to' just means that there is a 'to xxx' in the definition. In our case 'to cause any one to wish to arrange in order'.  
   * Note 2: According to current basicadjust and basicdisplay, `<div n="to"/>` will cause a line break,
      as do all types of `div` markup in mw.
* 'Desid. Caus.' is not one abbreviation but two; so we change markup to `<ab>Desid. Caus.</ab>` to
   be one abbreviation, so both words will be in bold.
* BUT now we need a tooltip for the new abbreviation `Desid. Caus.`, which is accomplished by 
   editing make_input.txt for mw  in csl-pywork/v02 ([See commit](https://github.com/sanskrit-lexicon/csl-pywork/commit/ffaef2de7e6d7b572d109abb604a74187566518f)).

There were 3 other 'Desid. Caus.' cases, which I marked similarly:
 L=4238, ADigA; 7382,anuSak; 91164,dah.

There was one case of 'Desid. Pass.' (L=9606, aparuD), which I changed similarly
